plugin/templates/chat_panel.md.jinja

```diff
@@ -58,21 +61,26 @@
     (
       "CODE_BLOCK_COMMANDS_" ~ index|string,
       (
+        "<div class='code-actions'>" ~
         "<a class='icon-link' href='" ~ command_url('copilot_conversation_copy_code', {"window_id": window_id, "code_block_index": index}) ~ "'>" ~
         "<img class='icon icon-link' src='" ~ asset_url('copy.png') ~ "' /></a>" ~
         "<span></span>" ~
         " <a class='icon-link' href='" ~ command_url('copilot_conversation_insert_code_shim', {"window_id": window_id, "code_block_index": index}) ~ "'>" ~
-        "<img class='icon icon-link' src='" ~ asset_url('insert.png') ~ "' /></a>\n\n"
+        "<img class='icon icon-link' src='" ~ asset_url('insert.png') ~ "' /></a>" ~
+        "</div>"
       ) | safe,
     )
 ) %}
 {% endfor %}
 {{ section.message | multi_replace(code_block_replacements) | safe }}
```

Comment:
Probably need an extra empty line here or somewhere. The second response in your screenshot doesn't look right. It didn't rendered as markdown correctly.

Reply:
this should be fixed now

Reply:
Plus, your icons looks better. Can they be publicly used?

Reply:
I think there are some local changes on your side which make this PR incomplete?

Reply:
- The broken icons issue is fixed now. I was testing some alternative icons and forgot to revert the file names before my second push.
- This PR now generates the following view on Linux (with the Mariana color scheme):

![image](https://github.com/user-attachments/assets/40918183-0d77-4fb2-a7e2-3e85f5761f38)

- I’ve made additional changes on my local system (e.g. moved icons to the right, relocated the thumbs-up/down icons below the response). I planned to push these progressively because I’m still gaining confidence with contributing via Git.
- I thought incremental changes would be easier to track, but let me know if you’d prefer a single push with all the updates.
- The icons are sourced from https://github.com/Remix-Design/RemixIcon